docs/http/traffic-policy/gallery.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Rule Gallery
+pagination_label: Rule Gallery
+---
+
+import ConfigExample from "/src/components/ConfigExample.tsx";
+
+# Rule Gallery
+
+Explore a curated collection of example configurations spanning from common to unconventional use-cases for the Traffic Policy module.
+
+## Deny non-GET requests:
+
+This rule denies all inbound traffic that is not a GET request.
+
+<ConfigExample
+	config={{
+		inbound: [
+			{
+				expressions: ["req.Method != GET"],
+				actions: [{ type: "deny" }],
+			},
+		],
+	}}
+/>
+
+## Custom response for unauthorized requests
+
+This rule sends a custom response with status code `401` and body `Unauthorized` for requests without an Authorization header.
+
+<ConfigExample
+	config={{
+		inbound: [
+			{
+				expressions: ["req.Headers['Authorization'] == ''"],
```

Comment:
This doesn't work — the value of `req.Headers['Authorization'] is a list of strings. Do you want `!hasReqHeder('Authorization')`?